wagtail/admin/panels/base.py

```diff
@@ -244,6 +247,7 @@ def is_required(self):
         def get_context_data(self, parent_context=None):
             context = super().get_context_data(parent_context)
             context["self"] = self
+            context["attrs"] = self.panel.attrs
```

Comment:
```suggestion
```

Let's remove this here, it seems like this is not the approach taken for other similar things in BoundPanel like icon, instead let's create a property like this...

```python
        @property
        def attrs(self):
            return self.panel.attrs
```

Maybe put it above or below the `def icon` on line ~235-ish. Let's try to keep the addition consistent, either we add attrs as the last thing or add it consistently with some context like before/after icon.